### ISSUE TITLE
Xiaomi mi a2 lite 3/32gb (codename: daisy)

### ISSUE THREAD

**Issue by patroldo** [Halium/projectmanagement#197]
Tree: 

- [x] Create manifest 
- [x] Boot image and system image build successfully
- [x] Device boots into rootfs, `usb: Manufacturer: GNU/Linux Device` appears in `dmesg` on host.
- [x] LXC container starts and does not crash
- [ ] libhybris tests
  * [ ] test_gps
  * [ ] test_hwcomposer
  * [ ] test_lights
  * [ ] test_vibrator
  * [ ] test_wifi
  * [ ] test_sensors
  * [ ] test_audio
  * [ ] test_camera
  * [ ] test_input
       _test_nfc - NFC not supported by device
  * [ ] test_recorder

**Comment by enesutku07** [comment]
please send file or link

**Comment by patroldo** [comment]
@enesutku07
[Manifest](https://github.com/Halium/halium-devices/blob/ee6ef92e1cf3bf383c85c7929292079b4c5a1a71/manifests/xiaomi_daisy.xml)

**Comment by paulopmp** [comment]
Can you send a tutorial for install the system in the my xiaomi mim a2 lite?

**Comment by patroldo** [comment]
> Can you send a tutorial for install the system in the my xiaomi mim a2 lite?

Yes, of course. I can send you necessary files, and instruction, how to install ubuntu touch with them. Of course it will be requiring clear device with unlocked bootloader. To do so, please, connect me via telegram @Chirick2051 :)

**Comment by marie-bnl** [comment]
If you have an installation tutorial, why not just post it here publicly?
